Source/Core/CommandLineOptions.cs

```diff
@@ -1822,7 +1822,7 @@ checking errors
                    s = debug statistics
                 0..9 = number of iterations before applying a widen (default=0)
   /noinfer      turn off the default inference, and overrides the /infer
-                switch on its left
+                switch on its left (default)
```

Comment:
Wait, why do we need `/noinfer` if the default behavior now is to not infer?
It seems to me that we do not need this switch any more.